picard/formats/mutagenext/tak.py

```diff
@@ -34,11 +34,11 @@
 
 try:
     from mutagen.tak import (
-        Open,
         TAK,
+        Open,
```

Comment:
This looks incorrect to me, why is it placed after `TAK`?

Reply:
Odd indeed, I'll look into it. But I assume it is because of `TAK` being all uppercase, and isort considers these constants and sorts them to the top, just like in https://github.com/metabrainz/picard/blob/master/picard/config.py#L43-L48

This is one of the places isort did not touch before, and hence this was manually sorted. I'll see if this can be prevented, but it might be easier to just live with this case.

Reply:
Yes, it is because of https://pycqa.github.io/isort/docs/configuration/options.html#order-by-type . `TAK` is all uppercase, so goes into the first group., followed by Open and TAKHeaderError which both qualify for the second camel-case group.

We can either add an `# isort:skip` here or just accept this case (I prefer the latter).